docs/governance/overview.md

```diff
@@ -2,29 +2,29 @@
 
----
 
-Holders of DCR can time-lock their funds in exchange for [tickets](../proof-of-stake/overview.md). The ticket price (the amount that must be time-locked) is adjusted dynamically every 144 blocks (~12 hrs), in an attempt to keep PoS subsidy returns stable over time. Tickets allow one to participate in Decred's governance in three ways, two on-chain and one off-chain. 
 
+---
 
-In each block, five live tickets are selected pseudo-randomly and called to vote on-chain. Tickets are called to vote after an average of around 28 days, once a ticket has voted the DCR which was time-locked to buy it matures (un-locks) after 256 blocks, along with a portion of the block reward.
+## On-chain voting
 
-On-chain voting serves the following purposes:
+On-chain voting serves two main purposes: validating blocks created by Proof-of-Work (PoW) miners and voting on proposed consensus rule changes. 
 
-1. [Consensus rule voting](consensus-rule-voting/overview.md) to *approve or reject a proposed change to the consensus rules* of the protocol. A proposed change must be approved by 75% of non-abstaining tickets to take effect.
+### Block voting
 
-1. Voting to *approve the work of PoW Miners*. In order for a PoW Miner to receive their share of the block reward, at least three of the five tickets called in the subsequent block must approve their block. This gives ticket-holders power over PoW Miners in the case of undesirable behavior by miners (e.g. mining empty blocks), although this power is yet to be exercised on mainnet.
+In each block, five tickets are selected pseudo-randomly and called to vote on-chain. It takes an average of 28 days for a ticket to be called to vote, though a ticket can be called anytime between 256 blocks after purchase (~20 hrs) to up to 40,960 blocks after purchase (~4 months). Tickets then vote to approve or reject the previous block of transactions created by a PoW miner. At least three out of five tickets called to vote must respond for the block to be considered valid. If a majority of voting tickets vote to approve the block, the block is accepted. If less than 50% of voting tickets approve the block (or there is a tie), all transactions from the previous block are removed and returned to the mempool; the PoW miner does not receive a reward. This power provides a check against malicious PoW miner behavior (e.g. mining empty blocks), as well as forming the basis of Decred’s [fork resistance](https://medium.com/decred/detailed-analysis-of-decred-fork-resistance-93022e0bcde7). After a ticket is called to vote, the DCR locked in that ticket is unlocked and returned to the purchaser’s wallet, along with the PoS reward if the ticket successfully voted. 
```

Comment:
> the PoW miner does not receive a reward.

Actually the miner does receive their reward, but it is taken away when the tickets vote to reject the block. The miner is in possession of the reward for one block, but they cannot spend it because of the coinbase maturity rules.

Reply:
Have reworded to:

> ; the PoW miner's reward is revoked. 

I'm tempted to further flesh this out, but don't want to go into too much detail on an intro page. I do think a separate page on block voting, as discussed in [this issue](https://github.com/decred/dcrdocs/issues/615), would be a good idea and a good place to put this detail.